s3/s3.go

```diff
@@ -94,6 +106,16 @@ func NewS3Client(opts S3ClientOpts) (S3Client, error) {
 		} else {
 			minioClient, err = minio.New(s3cli.Endpoint, s3cli.AccessKey, s3cli.SecretKey, s3cli.Secure)
 		}
```

Comment:
How come using envvar not SEClientOpts?

Reply:
I thought about that as well and figured this was more of the just works approach for the end user. The aws web pod identity webhook automatically injects those env vars into the containers when the service account has an IAM role applied to it. The other option would be to add some configuration to S3ClientOpts that is like useISRA: true then it would still end up having to use those ENV vars to pull the configuration.

Reply:
I suppose that is an unused parameter to the function now though, I can remove it as well.